lib/components/Highlighter.tsx

```diff
@@ -75,34 +57,21 @@ export const Highlighter = ({
     [highlighted, transitionDuration],
   );
 
-  return <HighlighterContext.Provider value={value}>{children}</HighlighterContext.Provider>;
+  return (
+    <>
+      <ScrollRestoration />
```

Comment:
As we just discussed in person, let's see if it works to put `<ScrollRestoration />` inside of `TRIApp`, since it's supposed to be a singleton.